client/api/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+import { useAuth } from '../store/users/AuthContext';
+
+export const useAPI = () => {
+  const auth = useAuth();
+  const instance = axios.create({ baseURL: '/api' });
```

Comment:
looks like we can get rid of PostURL now. If we use this for post and get requests I think we'll need anonymous tokens right?

Reply:
We don't need tokens at all for unprotected routes unless we want to track

Reply:
ok I get it so if we don't use the authMiddleware on the route then it wont matter if a token is passed or not.